wordpress-mobile/WordPress-Android
# Rename layout files

Parent #13328 

This PR is Part I of a consolidation effort of the Jetpack backup/restore features. There is a lot of overlap and the time is right to refactor where possible for sharing between the two. This first part streamlines the layout files by renaming, deleting, and then updating the import statements. Don't let the file count scare you, there were 8 deletions and all changes are in the import statements.

[x] Fragment Layout files
    - Replaced all individual fragments (listed below) with `jetpack_backup_restore_fragment`
        backup_download_complete_fragment
        backup_download_details_fragment
        backup_download_progress_fragment
        restore_complete_fragment
        restore_details_fragment
        restore_warning_fragment
        restore_progress_fragment

[x] List item layout files
    - Replaced restore_list_subheader_item with jetpack_backup_restore_list_subheader_item
    - Renamed backup_list_addition_information_item as jetpack_backup_restore_list_additional_information_item 
    - Renamed backup_list_progress_item as jetpack_list_progress_item 
    
[x] Activity layouts
    - Renamed backup_download_activity to jetpack_backup_restore_activity
    - Replaced restore_activity with jetpack_backup_restore_activity

**Note:**
- I did not make any style changes in this PR
- There is an unused id in `jetpack_backup_restore_fragment.xml`. I fixed this in Part III #13790 - slipped my line of site in this one.

**To test:**
- Run the app and make sure the backup/download still function as expected

PR submission checklist:
